packages/contracts-bedrock/src/dispute/weth/DelayedWETH.sol

```diff
@@ -87,14 +87,14 @@ contract DelayedWETH is OwnableUpgradeable, WETH98, IDelayedWETH, ISemver {
     /// @inheritdoc IDelayedWETH
     function recover(uint256 _wad) external {
         require(msg.sender == owner(), "DelayedWETH: not owner");
-        require(address(this).balance >= _wad, "DelayedWETH: insufficient balance");
-        payable(msg.sender).transfer(_wad);
+        uint256 amount = _wad < address(this).balance ? _wad : address(this).balance;
+        payable(msg.sender).transfer(amount);
```

Comment:
This seems reasonable, but why would the owner be recovering funds via this approach while the contract is unpaused and users are able to withdraw normally?

Reply:
This is mostly meant to be used when the system is paused in the case where a large number of dispute contracts are bugged and a single global withdrawal is the only solution (rather than withdrawing from specific dispute game contracts).

Reply:
But if the system is paused, how would someone withdraw small amounts?

Reply:
Ah yeah, good point. I think it's probably still worth having here in case there's some weird emergency edge-case where the optimal strategy is to withdraw first and pause later. Can't think of a good reason why this would be necessary but I can't see any real downside in having the function be easier to execute if needed.

Reply:
Yeah, I'm happy with the change, just checking my understanding.